pkgs/development/libraries/libgpg-error/default.nix

```diff
@@ -28,8 +28,11 @@ in stdenv.mkDerivation (rec {
     sed '/BUILD_TIMESTAMP=/s/=.*/=1970-01-01T00:01+0000/' -i ./configure
   '' + lib.optionalString (stdenv.hostPlatform.isAarch32 && stdenv.buildPlatform != stdenv.hostPlatform) ''
     ln -s lock-obj-pub.arm-unknown-linux-gnueabi.h src/syscfg/lock-obj-pub.linux-gnueabihf.h
-  '' + lib.optionalString stdenv.hostPlatform.isMusl ''
+  '' + lib.optionalString (stdenv.hostPlatform.isx86_64 && stdenv.hostPlatform.isMusl) ''
```

Comment:
What happens when `isMusl = true` and hostPlatform is not x86-64? (i686, aarch64, ...)

Previously they would match the `optionalString` and the link would have been created, but maybe it was unused.

Reply:
It is unused. The configure script only includes the header for the arch being built.